e2e/test-component/scenarios/embedding-sdk/collection-browser.cy.spec.tsx

```diff
@@ -0,0 +1,72 @@
+import { CollectionBrowser } from "@metabase/embedding-sdk-react";
+
+import { signInAsAdminAndEnableEmbeddingSdk } from "e2e/support/helpers/component-testing-sdk";
+import {
+  mockAuthProviderAndJwtSignIn,
+  mountSdkContent,
+} from "e2e/support/helpers/component-testing-sdk/component-embedding-sdk-helpers";
+import { getSdkRoot } from "e2e/support/helpers/e2e-embedding-sdk-helpers";
+
+describe("scenarios > embedding-sdk > collection browser", () => {
+  describe("personal collection", () => {
+    beforeEach(() => {
+      signInAsAdminAndEnableEmbeddingSdk();
+      cy.signOut();
+      mockAuthProviderAndJwtSignIn();
+
+      cy.intercept("GET", "/api/collection/*").as("getCollection");
+    });
+
+    ["personal", undefined].forEach(collectionId => {
+      it(`should show the personal collection when collectionId is ${collectionId ? collectionId : "not passed"}`, () => {
+        let personalCollectionId: number;
+
+        cy.intercept("GET", "/api/user/current", req => {
+          req.continue(res => {
+            personalCollectionId = res.body.personal_collection_id;
+          });
```

Comment:
(nit: I think we could also get the collection id via `cy.get("@getCurrentUser").then(res => res.body.personal_collection_id)` as we already saved it via `.as()`)